util.js

```diff
@@ -48,4 +48,19 @@ exports.next = function (createStream, opts, property, range) {
 }
 
 
+exports.ScrollNotify = function (div, scroller) {
+  var cl = 'hypertabs--notify'
+  scroller.observ(function (len) {
+    if(div.classList.contains(cl) != (len != 0)) {
+      if(len) div.classList.add(cl)
+      else div.classList.remove(cl)
+    }
```

Comment:
**[curiosity]**: I honestly can't see how this works, but can see it working in the browser.
Here `div` is the public scroller container. I can see `.hypertabs--notify` getting added to that. 

It's also getting added to the tab, class `.hypertabs__tab`.
What wizardry is this?